parsl/executors/high_throughput/interchange.py

```diff
@@ -318,10 +325,7 @@ def start(self, poll_period=1):
         interesting_managers = set()
 
         while not self._kill_event.is_set():
-            logger.debug("[MAIN] starting one main loop iteration")
-            logger.debug("[MAIN] entering poll")
-            self.socks = dict(poller.poll(timeout=poll_period))
-            logger.debug("[MAIN] leaving poll")
+            self.socks = dict(poller.poll(timeout=self.poll_period))
```

Comment:
I think it should be `poll_period` rather than `self.poll_period` here. Otherwise, it is meaningless to set `poll_period` in the beginning `start` function.

Reply:
Fixed.